### PR TITLE
[FEAT / FIX] Add livl product and other behaviour

### LINKED ISSUE #38: Translate product names

### LINKED ISSUE #39: Create script to facillitate the prefab import
This script should apply the correct tag, layer, mesh collider and product info to all the child of a prefab !

### PR DESCRIPTION
**Pull request by Augustin68**
### What has changed ?

- Add new custom livl products
- Fixed their spawn position height
- Translate their name to make the game easier to play
- Facilitate DX with a script for livl product prefabs

### How to test ?
- Just start the game.
- You must be able to scan any items
- All the items should be named in french in your livl app 🙌

close #38 
close #39

### CODE REVIEW

File: Assets/Scripts/PlayerShoot.cs [hunk 1]
```diff
@@ -51,25 +51,34 @@ private void Update()
 
     private void Shoot()
     {
+        Debug.Log("SHOOT !");
```
Comment (Augustin68) [inline Assets/Scripts/PlayerShoot.cs:54]:
```suggestion
```

File: Assets/Scripts/PlayerShoot.cs [hunk 2]
```diff
@@ -51,25 +51,34 @@ private void Update()
...
         RaycastHit hit;
         if (Physics.Raycast(cam.transform.position, cam.transform.forward, out hit, weapon.range, mask))
         {
+            Debug.Log("hit !");
```
Comment (Augustin68) [inline Assets/Scripts/PlayerShoot.cs:58]:
```suggestion
```

File: Assets/Scripts/StoreItem.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class StoreItem : MonoBehaviour
+{
+    //[SerializeField]
+    //[Tooltip("The id of the item")]
+    //public int id;
```
Comment (Augustin68) [inline Assets/Scripts/StoreItem.cs:9]:
```suggestion
```

File: Assets/Scripts/PlayerShoot.cs (hunk 2 above)
Comment (Augustin68) [inline Assets/Scripts/PlayerShoot.cs]:
```suggestion
        RaycastHit hit;
        if (Physics.Raycast(cam.transform.position, cam.transform.forward, out hit, weapon.range, mask))
        {
```

**Review by jvondermarck — APPROVED** [review]
Great work !! Everything is fine for me !

**Pull request merged**